src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/spreadsheet/PricesSpreadsheetGenerator.kt

```diff
@@ -36,23 +37,33 @@ class PricesSpreadsheetGenerator(
       val journeys = journeyService.distinctJourneysIncludingPriced(supplier, startDate)
       val summaries = moveService.moveTypeSummaries(supplier, startDate)
 
-      StandardMovesSheet(workbook, header).also { logger.info("Adding standard prices.") }.apply { writeMoves(moves[0]) }
+      StandardMovesSheet(workbook, header).also { logger.info("Adding standard prices.") }
+        .apply { writeMoves(moves[0]) }
 
-      RedirectionMovesSheet(workbook, header).also { logger.info("Adding redirect prices.") }.apply { writeMoves(moves[1]) }
+      RedirectionMovesSheet(workbook, header).also { logger.info("Adding redirect prices.") }
+        .apply { writeMoves(moves[1]) }
 
-      LongHaulMovesSheet(workbook, header).also { logger.info("Adding long haul prices.") }.apply { writeMoves(moves[2]) }
+      LongHaulMovesSheet(workbook, header).also { logger.info("Adding long haul prices.") }
+        .apply { writeMoves(moves[2]) }
 
       LockoutMovesSheet(workbook, header).also { logger.info("Adding lockout prices.") }.apply { writeMoves(moves[3]) }
 
-      MultiTypeMovesSheet(workbook, header).also { logger.info("Adding multi-type prices.") }.apply { writeMoves(moves[4]) }
+      MultiTypeMovesSheet(workbook, header).also { logger.info("Adding multi-type prices.") }
+        .apply { writeMoves(moves[4]) }
 
-      CancelledMovesSheet(workbook, header).also { logger.info("Adding cancelled moves.") }.apply { writeMoves(moves[5]) }
+      CancelledMovesSheet(workbook, header).also { logger.info("Adding cancelled moves.") }
+        .apply { writeMoves(moves[5]) }
 
       JourneysSheet(workbook, header).also { logger.info("Adding journeys.") }.apply { writeJourneys(journeys) }
 
       SummarySheet(workbook, header).also { logger.info("Adding summaries.") }.apply { writeSummaries(summaries) }
 
-      SupplierPricesSheet(workbook, header).also { logger.info("Adding prices used for supplier $supplier.") }.apply { writePrices(supplierPrices.get(supplier)) }
+      SupplierPricesSheet(
+        workbook,
+        header
+      ).also {
+        logger.info("Adding $supplier prices for effective year ${effectiveYearForDate(startDate)}.")
+      }.apply { writePrices(supplierPrices.get(supplier, effectiveYearForDate(startDate))) }
```

Comment:
Ensures we are using the correct effective years prices when putting into generated spreadsheet.